fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pytest = "*"
+pytest-html = "*"
+pytest-metadata = "*"
+requests = "*"
+black = "*"
+flake8 = "*"
+
```

Comment:
I'd drop `black` and `flake8`

Reply:
Whoops missed the reply. I would just omit anything that's not absolutely needed to run these.

Reply:
Let me move them under dev packages, I'd like to use them